pkg/controller/builder.go

```diff
@@ -33,10 +36,11 @@ type Builder struct {
 	ControllerContext *controllercmd.ControllerContext
 }
 
-func (b *Builder) BuildControllers(subDirectories ...string) (map[string]factory.Controller, map[string]factory.Controller, []configv1.ObjectReference, error) {
+func (b *Builder) BuildControllers(subDirectories ...string) (map[string]factory.Controller, map[string]factory.Controller, map[string]factory.Controller, []configv1.ObjectReference, error) {
```

Comment:
We should add go doc for the method. Specially how we are starting new controller for each default catalog.